linera-base/src/crypto.rs

```diff
@@ -167,7 +166,7 @@ impl Serialize for CryptoHash {
         if serializer.is_human_readable() {
             serializer.serialize_str(&self.to_string())
         } else {
-            serializer.serialize_newtype_struct("CryptoHash", &self.0)
+            serializer.serialize_newtype_struct("CryptoHash", &self.0 .0)
```

Comment:
Maybe `CryptoHash` deserves an `as_bytes` method to avoid the `.0 .0`.

Reply:
It already has `as_bytes` but it will be `as_bytes().0` anyway (the second `.0` is for extracting `[u8; N]` from `FixedBytes`.